libs/contrib/Data/Always.idr

```diff
@@ -0,0 +1,39 @@
+module Data.Always
```

Comment:
Shouldn't this be in Data.List or Data.List.All/Data.List.Always? Similar structures could be defined for lots of list-like structures, such as Vect and friends.